ibm/mas_devops/roles/ocp_provision/defaults/main.yml

```diff
@@ -84,6 +84,7 @@ ipi_compute_replicas: "{{ lookup('env', 'IPI_COMPUTE_REPLICAS') | default('3', t
 # For now we can default the type to the best fit for AWS as that's all we support atm
 ipi_controlplane_type: "{{ lookup('env', 'IPI_CONTROLPLANE_TYPE') | default('m5.4xlarge', true) }}"
 ipi_controlplane_replicas: "{{ lookup('env', 'IPI_CONTROLPLANE_REPLICAS') | default('3', true) }}"
+ipi_storage_size: "{{ lookup('env', 'IPI_STORAGE_SIZE') | default('300', true) }}"
```

Comment:
By defaulting this to 300 we are setting this value for 99% of customers who will not customize this ... is that really the intent here?  What value would this normally be if not explicitly set?

Suggest it's better to not a default for this, unless someone sets this we continue to function as we did before, by not setting `rootVolume.size` .. also suggest calling it `ipi_rootvolume_size` so it's more reflective of it's use.  Also, this needs to be added to the documentation in the role's `README.md`